CHANGELOG

```diff
@@ -1,3 +1,6 @@
 1.0.0-b6
 
+* Use SFINAE on return values
+* Use beast::error_code instead of nested types
```

Comment:
I'm not sure about your intention for squashing but, at least for this iteration, it appears that using `beast::error_code` doesn't happen until the next commit.  If they get squashed it really doesn't matter.  Just FYI.